src/collective/ifttt/actions/ifttt.py

```diff
@@ -88,25 +91,35 @@ class IftttTriggerActionExecutor(object):
     """
     timeout = 120
 
-    def __init__(self, element):
+    def __init__(self, element, context, event):
         self.element = element
+        self.context = context
+        self.event = event
 
     def __call__(self, *args, **kwargs):
-        r = self.element.request
-        ifttt_name = self.element.ifttt_name
+        Title = self.element.Title
+        URL = self.element.URL
         payload_option = self.element.payload_option
-        payload = self.element.payload
+        # TODO explore context and event; element is defined with Ifttt Trigger form
+        # TODO define ifttt_url with secret key
+        ifttt_name = self.event.title
```

Comment:
ifttt_name should be configured in the content rule configuration schema and is read from the element (it was correct before this commit)